test/parallel/test-stream2-writable.js

```diff
@@ -328,7 +328,7 @@ test('dont end while writing', function(t) {
     assert(!this.writing);
     wrote = true;
     this.writing = true;
-    setTimeout(function() {
+    setImmediate(function() {
```

Comment:
there is a _slight_ difference between `setTimeout` and `setImmediate`, given where they are executed in the event loop. If it is indifferent for the test, e.g. even `process.nextTick` is fine, then add it as a comment.